docs/releases.rst

```diff
@@ -36,6 +27,42 @@ contrast to the :ref:`/spec/core/contact` key, this field is not
 supposed to be updated and can be useful when trying to track down
 the original author for consultation.
 
+Add support for running playbooks from Ansible collections specified
+using the ``namespace.collection.playbook`` notation.
+
+Added ``--dry`` option for the ``beaker`` provision plugin. When
+used it prints the Beaker Job XML without submitting it.
+
+`Results specification documentation`__ has now a dedicated place
```

Comment:
The link should use an internal reference instead of referring to an absolute URL - Sphinx will take care of rendering the right URL.

```
:ref:`/spec/results` now has a dedicated place...
```

Reply:
ty

Reply:
changed